engine/src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -890,4 +880,14 @@ class SlickDataAccess(databaseConfig: Config) extends DataAccess {
 
     runTransaction(action)
   }
+
+  private def materializeWorkflowDescriptor(id: WorkflowId, workflowSourceFiles: WorkflowSourceFiles)(implicit ec: ExecutionContext): Future[WorkflowDescriptor] = {
```

Comment:
Hmm I'm not sure I understand, if `WorkflowDescriptorMaterializerActor` is meant to be a singleton throughout Cromwell, then accessing it shouldn't be a problem. If it's not supposed to be a singleton and anyone can pop up its own, then it shouldn't be a problem either to create one from the WorkflowActor.
Or am I missing something ?